proto/astarteplatform/msghub/property.proto

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of Astarte.
+ *
+ * Copyright 2024 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+syntax = "proto3";
+
+package astarteplatform.msghub;
+
+import "astarteplatform/msghub/astarte_type.proto";
+import "astarteplatform/msghub/interface.proto";
+
+/* A message representing the property value associated to a certain interface and path. */
+message Property {
+  string path = 1;                      // Property path.
+  AstarteDataTypeIndividual value = 2;  // Astarte data.
```

Comment:
If we make this field optional, maybe we should create two messages. One for the call to the single property, which is optional (and maybe without the path). Another one for the other two calls, which has the value repeated and will never return null properties

Reply:
Even when the all_properties method is called we should return unsetted values I think. At least this is how it is handled in the sdk method for the various store implementations ([memory store](https://github.com/astarte-platform/astarte-device-sdk-rust/blob/e8469c09100be2ad5446a7eb3bee359ee8314182/src/store/memory.rs#L153)). So it seems like we can get away with using only the Property type.

Reply:
As we cleared IRL the unset property are only stored until they are sent. The user should never see them when getting all properties.